### PR TITLE
Add some content to the read me

### PR DESCRIPTION
**Pull request by theramis**
More peeps have been asking me about where the source is so thought I'd clean up the read me a tad. 

Best way to see the readme is at this link. 
https://github.com/theramis/aucvid/tree/readme

### CODE REVIEW

**Review by andycarrell — APPROVED** [review]
Nice! Super concise and easy to read

File: .github/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  <br />
+  <a href="https://github.com/theramis/aucvid/issues/new?assignees=&labels=bug&template=bug_report.md&title=%F0%9F%90%9B%20Bug:%20">Report a Bug</a>
+  ·
+  <a href="https://github.com/theramis/aucvid/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=%F0%9F%92%A1%20Feature:%20">Request a Feature</a>
+</div>
+<div align="center">
+<br />
+
+[![license](https://img.shields.io/github/license/theramis/aucvid.svg?style=for-the-badge)](../LICENSE)
+![PRs welcome](https://img.shields.io/badge/PRs-welcome-ff69b4.svg?style=for-the-badge)
```
Comment (jishaal) [inline .github/README.md:18]: Is this something we are doing? I feel like we are we may need to put little bit more detail on getting it up and running e.g. certain requirements currently to have github auth token (might be it?)

Comment (theramis) [reply .github/README.md:18]: yea fair, I may or may not have copy pasted this readme from somewhere else 😂 

Will remove

Comment (jishaal) [reply .github/README.md:18]: Sweet ta!

**Review by jishaal — APPROVED** [review]

**Pull request merged**